app/src/androidTest/java/com/github/fribourgsdp/radio/PlaylistRecyclerViewTest.kt

```diff
@@ -21,9 +21,6 @@ import java.util.concurrent.TimeUnit
 
 @RunWith(AndroidJUnit4::class)
 class PlaylistRecyclerViewTest {
-    @get:Rule
-    var activityRule = ActivityScenarioRule(UserProfileActivity::class.java)
-    private lateinit var user: User
```

Comment:
Why are you deleting this?

Reply:
Because if we are using the rule and then launching the activity manually, it launches the activity twice, which slows up the execution of the test and maybe also make it less stable.